src/SFA.DAS.EAS.Employer_Account.Database/Tables/JobHistory.sql

```diff
@@ -0,0 +1,5 @@
+﻿CREATE TABLE [dbo].[JobHistory]
+(
+	[Job] NVARCHAR(50) NOT NULL PRIMARY KEY, 
```

Comment:
Having `Job` as the primary key means that the `JobHistory` table can only store a single run for each particular job.
Even though the `PopulateAccountUsersInCollectionJob` job is only meant to be run once, allowing the recording of multiple runs for a job would make the `JobHistory` table more useful for reuse.

Also, `NVARCHAR` isn't a great data type for a primary key. Would it be better to have a numeric identity column as the primary key?

Reply:
Totally agree, will add an auto key and make the job name a secondary index.